### PR TITLE
Fixed performance issues related to cover image loading

### PR DESCRIPTION
**Pull request by paulijar**
The UI performance was absolutely horrible when the library contained lots of albums with embedded cover art. The biggest problem was that all the cover art was reloaded each time when a track or album or artist was selected for playing. The playing doesn't start before the covers are loaded. Another problem was that each cover picture was actually loaded twice.

### CODE REVIEW

**Comment by Faldon** [comment]
These code changes looks good to me. 
The Travis checks seems to be unrelated to them. (OwnCloud 7.1 does not work with php 5.4 e.g.)
@gnarula Your comments?

File: js/app/directives/albumart.js [hunk 1]
```diff
@@ -20,45 +20,57 @@
...
+	function setCoverImage(element, imageUrl) {
+		// remove placeholder stuff
+		element.html('');
+		element.css('background-color', '');
+		// add background image
+		element.css('filter', "progid:DXImageTransform.Microsoft.AlphaImageLoader(src='" + imageUrl + "', sizingMethod='scale')");
+		element.css('-ms-filter', "progid:DXImageTransform.Microsoft.AlphaImageLoader(src='" + imageUrl + "', sizingMethod='scale')");
+		element.css('background-image', 'url(' + imageUrl + ')');
+	}
+
```
Comment (gnarula) [inline js/app/directives/albumart.js]: trailing whitespace

**Comment by gnarula** [comment]
Thank you for your PR :) looks really good!There are a few trailing whitespaces in the file I've commented on. Once that's fixed it's good to merge I think :)

**Comment by paulijar** [comment]
So @gnarula, is this good to merge now? I will have some new PRs coming in once this one is merged.

**Comment by gnarula** [comment]
Yup, looks good! Keep em coming :)

**Comment by mmattel** [comment]
:+1:

**Pull request merged**